Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+def prLabels = getPullRequestLabels {}
+
+if ( prLabels.contains("test/integration") ) {
+
+  stage('Trust') {
+    enforceTrustedApproval('aerogear')
+  }
+
+  node ('osx') {
+    stage('Prepare osx workspace') {                    
+      // Clean workspace
+      deleteDir()
+
+      // Checkout to osx slave
+      checkout scm
+
+      // Clone example app
+      sh 'git clone git@github.com:aerogear/cordova-showcase-template.git cordova-example'
+
+      // Restart appium
+      sh """
+        lsof -i tcp:4723 | grep LISTEN | awk '{print \$2}' | xargs kill
+        nohup appium &>"\$HOME/appium.log" </dev/null &
+      """
+    }
+
+    stage ('Install dependencies') {
+      sh '''
+        npm install
+        npm run bootstrap
+        npm run build
+      '''
+      dir('cordova-example') {
+        sh '''
+          npm install
+          npm link ../packages/core
```

Comment:
We should about this IMO, we should test the available packages in NPM. We could do a pre-release before doing integration, then the full release.

Reply:
@josemigallas Absolutely not. This is for testing PR's with changes that are not released. If we going to change purpose of this test then we can change that.

Reply:
OK